cache/cache.go

```diff
@@ -0,0 +1,27 @@
+package cache
+
+import (
+	xcodecache "github.com/bitrise-io/go-xcode/xcodecache"
+)
+
+// Cache ...
+type Cache interface {
```

Comment:
How about calling this `SwiftPackageCache` to be more specific and avoid stuttering?